src/Lecture1.hs

```diff
@@ -67,7 +69,8 @@ sumOfSquares x y = error "TODO!"
 
 -}
 -- DON'T FORGET TO SPECIFY THE TYPE IN HERE
-lastDigit n = error "lastDigit: Not implemented!"
+lastDigit :: Int -> Int
+lastDigit n = mod (abs n) 10
```

Comment:
Absolutely correct 👍🏻 
Alternatively, you can use the infix form of a function by putting it inside backticks :slightly_smiling_face: 
```suggestion
lastDigit n = abs n `mod` 10
```

Reply:
Normal function application (aka ` ` space) has the highest precedence.

In Haskell, you can actually specify custom precedence for the function application in backticks (and `mod` has it).

In this particular case,

```haskell
abs n `mod` 10
```

may look more natural if you're coming from other popular languages, where mod is e.g. a `%` operator and you'd write this code like this:

```
abs(n) % 10
```